demoextendsymfonyform3/demoextendsymfonyform3.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+
+use DemoCQRSHooksUsage\Domain\Reviewer\Command\UpdateIsAllowedToReviewCommand;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\CannotCreateReviewerException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\CannotToggleAllowedToReviewStatusException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\ReviewerException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Query\GetReviewerSettingsForForm;
+use DemoCQRSHooksUsage\Domain\Reviewer\QueryResult\ReviewerSettingsForForm;
+use Doctrine\DBAL\Query\QueryBuilder;
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Exception\CustomerException;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ToggleColumn;
+use PrestaShop\PrestaShop\Core\Grid\Definition\GridDefinitionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Search\Filters\CustomerFilters;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\YesAndNoChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+
+/**
+ * Class DemoExtendSymfonyForm1 demonstrates the usage of CQRS pattern and hooks.
+ */
+class DemoExtendSymfonyForm3 extends Module
+{
+    public function __construct()
+    {
+        $this->name = 'demoextendsymfonyform3';
+        $this->version = '1.0.0';
+        $this->author = 'Mathieu Ferment';
+        $this->need_instance = 0;
+
+        parent::__construct();
+
+        $this->displayName = $this->getTranslator()->trans(
```

Comment:
```suggestion
        $this->displayName = $this->trans(
```

no need to get translator here, there's shorter `trans` method in `Module`